### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.resource.resources.v2017_05_10.models.resource_management_error_with_details.ResourceManagementErrorWithDetails
  fullName: azure.mgmt.resource.resources.v2017_05_10.models.resource_management_error_with_details
  langs:
  - python
  module: azure.mgmt.resource.resources.v2017_05_10.models.resource_management_error_with_details
  name: resource_management_error_with_details
  source:
    id: resource_management_error_with_details
    path: azure-mgmt-resource\azure\mgmt\resource\resources\v2017_05_10\models\resource_management_error_with_details.py
    remote:
      branch: master
      path: azure-mgmt-resource\azure\mgmt\resource\resources\v2017_05_10\models\resource_management_error_with_details.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.mgmt.resource.resources.v2017_05_10.models.resource_management_error_with_details
references:
- fullName: azure.mgmt.resource.resources.v2017_05_10.models.resource_management_error_with_details.ResourceManagementErrorWithDetails
  isExternal: false
  name: ResourceManagementErrorWithDetails
  parent: azure.mgmt.resource.resources.v2017_05_10.models.resource_management_error_with_details
  uid: azure.mgmt.resource.resources.v2017_05_10.models.resource_management_error_with_details.ResourceManagementErrorWithDetails
